test/python/utest_masarclient/test_alarm.py

```diff
@@ -0,0 +1,108 @@
+from masarclient.alarm import Alarm as Alarm
+import unittest
+from types import *
+
+
+'''
+
+Unittests for masarService/python/masarclient/alarm.py
+
+'''
+
+
+class TestAlarm(unittest.TestCase):
+
+    '''
+    Create test values for each test. Uses default constructor values.
+    '''
+    def setUp(self):
+        self.test_alarm = Alarm()
+
+    '''
+    Ensures test_alarm value is cleared after each test
+    '''
+    def tearDown(self):
+        self.test_alarm = None
```

Comment:
If this is necessary then there is some problem with ref. counting in the c++ code.